README.md

```diff
@@ -84,7 +87,9 @@ Note that we use our `types` helper to define types in the result, and the `para
 Then, convert the JS Object to GraphQL (string) with `graphqlify`:
 
 ```typescript
-const gqlString = graphqlify.query('getUser', getUserQuery)
+import { query } from 'typed-graphqlify'
```

Comment:
I added these `import` statement to examples, because a lot of exports available now, and query examples are document in fact.

Reply:
That's probably for the best, it's clearer to people looking over it quickly